### ISSUE TITLE
Could not process xlsm document

### ISSUE THREAD

**Issue by blcham** [kbss-cvut/s-pipes#225]
When I tried to process xlsm document I got the following error :
 org.apache.poi.poifs.filesystem.OfficeXmlFileException: The supplied data appears to be in the Office 2007+ XML. You are calling the part of POI that deals with OLE2 Office Documents. You need to call a different part of POI to process this data (eg XSSF instead of HSSF)
	at org.apache.poi.poifs.storage.HeaderBlock.<init>(HeaderBlock.java:133)
	at org.apache.poi.poifs.storage.HeaderBlock.<init>(HeaderBlock.java:106)

**Comment by blcham** [comment]
xlsm files is confidential thus try to find another one first please.

**Comment by rodionnv** [comment]
Fixed in #222. Now the type of the excel file should be set explicitly:
```
* - "application/vnd.ms-excel" - EXCEL (XLS) file.
* - "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" - EXCEL (XLSX) file.
* - "application/vnd.ms-excel.sheet.macroEnabled.12" - EXCEL (XLSM) file.
```

**Comment by blcham** [comment]
@rodionnv, Good job! 

Moreover, can we autodetect it somehow if the type is not provided? (we would not do it now but later as enhancement of the module)
